lib/oneview-sdk/resource/api200/logical_switch.rb

```diff
@@ -46,12 +46,13 @@ def create
         self
       end
 
-      # Updates this object using the data that exists on OneView
-      # @note Will overwrite any data that differs from OneView
-      # @return [Resource] self
-      def refresh
+      # Reclaims the top-of-rack switches in a logical switch.
+      # @return [OneviewSDK::LogicalSwitch] self
+      def refresh_state!
```

Comment:
But, the return of this method is a LogicalSwitch with associated data (Switches uris and SwitchGroup uris, etc).
And the http verb of request used in this method is the PUT (like documentation describes), I imagine that server can change the data of LogicalSwitch when applyes the refresh.
Please, tell me if I am with wrong undestanding. 
What are you think?